lib/commands/jukebox/Jukebox.js

```diff
@@ -407,6 +413,25 @@ class Jukebox {
                 message.reply('cannot delete the current song.')
             })
     }
+
+    replay(context) {
+        const { message } = context
+
+        if (!this.nowPlaying) {
```

Comment:
i don't think this is strictly necessary, if replay is enabled it will repeat the next playing song after it completes for the first time? @forestbelton is this favorable ?

in addition, how should replay work with stopping the jukebox ? should it be reset to false